File: Nível 1/1116 - Dividindo X por Y/main.py
quantia = int(input("A: "))

for i in range(1, quantia + 1):
    valor = input("B: ")

    x1,x2 = valor.split(" ")

    x1 = float(x1)
    x2 = float(x2)

    if (x2 == 0):
        print("divisao impossivel")
    else:
        calc = x1 / x2 
        print(calc)

File: Nível 1/1116 - Dividindo X por Y/main.lua
quantia = tonumber(io.read())

for i = 1, quantia do 
    valor = io.read()
    x1, x2 = valor:match("(%S+)%s+(%S+)")
    x1, x2 = tonumber(x1), tonumber(x2)

    if x2 == 0 then
        print("divisao impossivel")
    else
        calc = x1 / x2 
        print(calc)
    end
end